website/content/docs/configuration/storage/index.mdx

```diff
@@ -40,3 +40,60 @@ storage "file" {
 For configuration options which also read an environment variable, the
 environment variable will take precedence over values in the configuration
 file.
+
+## Embedded storage vs. external storage
```

Comment:
Would you mind adding a link to this page from the storage concepts page to help with discoverability?

Reply:
Added a call-out note to reference from the concept page. ([commit](https://github.com/hashicorp/vault/pull/13999/commits/7e5b9ec3fd8e7f8afb1c4986e72be10c139953ac))

![callout-note](https://user-images.githubusercontent.com/7660718/153445289-c9911cdd-ac5f-4e62-a8ad-41cc4dd7b518.png)

Reply:
Small suggestion to consider, but I'd recommend making integrated storage the explicit (or at least implicit) recommendation here, and using the link for more details. This allows someone to understand what we recommend quickly without needing to navigate to and read a separate page (unless they want/need more details).

## Supported Storage Backends

For enterprise customers, HashiCorp provides official support for the following storage backends:
- [Vault's integrated storage](link): the recommended backend for most use cases, available in Vault 1.4 or later
- [Consul storage backend](link): leverage the key-value store of Consul, a multi-cloud service networking
   solution including service mesh, service discovery, and network infrastructure automation

Many options are available with community support, including the above. See our [Storage Configuration](link) section for more information.

[ insert your existing "Choosing a storage backend" call-out ]